R/derive_param_extreme_record.R

```diff
@@ -0,0 +1,233 @@
+#' Adds a Parameter based on first or last record from multiple sources
```

Comment:
```suggestion
#' Adds a Parameter Based on First or Last Record from Multiple Sources
```
We've been consistently capitalizing most words that aren't articles in our function titiles.